Summary Table/Summary Table Dev Flextable.Rmd

```diff
@@ -0,0 +1,398 @@
+
+time <- c(
+  latest_period_alcohol_hosp,
+  latest_period_alcohol_deaths,
+  str_wrap(latest_period_drug_hosp, 9),
+  latest_period_bowel_screening
+)
+
+# Values
+loc_values <- round_half_up(c(alcohol_hosp_latest, alcohol_deaths_latest, drug_hosp_latest, bowel_screening_latest), 1)
+other_loc_values <- bind_rows(other_locs_alcohol_hosp, other_locs_alcohol_deaths, other_locs_drug_hosp, other_locs_bowel_screening)
+hscp_values <- c(hscp_alcohol_hosp, hscp_alcohol_deaths, hscp_drug_hosp, hscp_bowel_screening)
+scot_values <- c(scot_alcohol_hosp, scot_alcohol_deaths, scot_drug_hosp, scot_bowel_screening)
+
+
+lifestyle_values <-data.frame(indicators, datatype, time, loc_values, other_loc_values,
+                          hscp_values, scot_values)
+
+
+colnames(lifestyle_values) <- headings
+
+lifestyle_table <- flextable(lifestyle_values) %>%
+  bold(part = "header") %>%
+  bold(j = 1, bold = TRUE, part = "body") %>%
+  fontsize(part = "header", size = 12) %>%
+  font(part = "body", fontname = "Arial") |>
+  font(part = "header", fontname = "Arial") |> 
+  color(part = "header", color = "white") %>%
+  bg(part = "header", bg = "#3F3685") %>%
+  bg(j = 1:3, part = "body", bg = "grey90") |>
+ # width(j = 1, width = "22em") |> 
+  border(border = fp_border(color = "white", width = 1), part = "body") |> 
+  set_table_properties(layout = "autofit")
+  
+lifestyle_table
+
+
+
+```
+
+
+
+## Hopsital and Community Care Summary Table
+
+
+```{r echo=FALSE}
+
+indicators <- c(
+  "Emergency admissions per 100,000",
+  "Unscheduled bed days per 100,000",
+  "A&E attendances per 100,000",
+  "Delayed discharges (65+) per 100,000",
+  "Potentially Preventable Admissions per 100,000"
+)
+datatype <- c("rate", "rate", "rate", "rate", "rate")
+time <- c(max_fy, max_fy, max_fy, max_fy, max_fy)
+
+# Values
+loc_values <- c(latest_emergency_adm_loc, latest_bed_days_loc, latest_ae_att_loc, latest_dd_loc, latest_ppa_loc)
+other_loc_values <- bind_rows(other_loc_emergency_adm, other_loc_bed_days, other_loc_ae_att, other_loc_dd, other_loc_ppa)
+hscp_values <- c(hscp_emergency_adm, hscp_bed_days, hscp_ae_att, hscp_dd, hscp_ppa)
+scot_values <- c(scot_emergency_adm, scot_bed_days, scot_ae_att, scot_dd, scot_ppa)
+
+
+UC_values <-data.frame(indicators, datatype, time, loc_values, other_loc_values,
+                          hscp_values, scot_values)
+
+colnames(UC_values) <- headings
+
+UC_table <- flextable(UC_values) %>%
+  bold(part = "header") %>%
+  bold(j = 1, bold = TRUE, part = "body") %>%
+  fontsize(part = "header", size = 12) %>%
+  font(part = "body", fontname = "Arial") |>
+  font(part = "header", fontname = "Arial") |> 
+  color(part = "header", color = "white") %>%
+  bg(part = "header", bg = "#3F3685") %>%
+  bg(j = 1:3, part = "body", bg = "grey90") |>
+ # width(j = 1, width = "22em") |> 
+  border(border = fp_border(color = "white", width = 1), part = "body") |>
+  set_table_properties(layout = "autofit")
+  
+UC_table
```

Comment:
Double check that `UC_values` and/or `UC_table` isn't used elsewhere before applying this suggestion.

```suggestion
uc_values <- tibble(indicators, datatype, time, loc_values, other_loc_values,
                          hscp_values, scot_values)

colnames(uc_values) <- headings

UC_table <- flextable(uc_values) %>%
  bold(part = "header") %>%
  bold(j = 1, bold = TRUE, part = "body") %>%
  fontsize(part = "header", size = 12) %>%
  font(part = "body", fontname = "Arial") |>
  font(part = "header", fontname = "Arial") |> 
  color(part = "header", color = "white") %>%
  bg(part = "header", bg = "#3F3685") %>%
  bg(j = 1:3, part = "body", bg = "grey90") |>
  border(border = fp_border(color = "white", width = 1), part = "body") |>
  set_table_properties(layout = "autofit")
  
uc_table
```